### PR TITLE
Feature/farhan

### PR DESCRIPTION
**Pull request by Farhanramadhana**

### CODE REVIEW

File: modules/farhan/models/index.js [hunk 1]
```diff
@@ -6,11 +6,46 @@ class FarhanModel{
...
+    if(minAge){
+      query += ` AND age >= ${minAge}`
+    }
+
+    if(maxAge){
+      query += ` AND age <= ${maxAge}`
+    }
+
+    if(search){
+      query += ` AND name like '%${search}%'`
```
Comment (wlisrausr) [inline modules/farhan/models/index.js]: `like` is part of query. Use an uppercase word for this. Change `like` to `LIKE`

File: modules/farhan/models/index.js [hunk 2]
```diff
@@ -6,11 +6,46 @@ class FarhanModel{
...
+    }
+
+    query += ` ORDER BY ${sort_by} ${order} LIMIT ${offset}, ${limit}`
+    console.log(query)
```
Comment (wlisrausr) [inline modules/farhan/models/index.js]: Don't forget to remove `console.log`.

File: modules/farhan/services/index.js [hunk 3]
```diff
@@ -20,9 +20,36 @@ class FarhanService{
 		};
 	}
 
-	async index(){
-		return await this.farhanModel.index()
-	}
+	async index(query){
```
Comment (wlisrausr) [inline modules/farhan/services/index.js]: Make sure the indentation in this async function is correct. Revise this part.

**Review by wndisra — CHANGES_REQUESTED** [review]
Already commented on every part that need to be change. Please revise it.

**Review by wndisra — APPROVED** [review]

**Pull request merged**